dataengineeringutils/glue.py

```diff
@@ -281,6 +282,40 @@ def metadata_folder_to_database(folder_path, delete_db = True, db_suffix = None)
         populate_glue_catalogue_from_metadata(table_metadata, db_metadata, check_existence=False)
 
 
+def glue_job_dir_to_s3(local_glue_jobs_dir, s3_glue_jobs_dir, include_folders = None, exclude_folders = None) :
```

Comment:
Is there a clearer name that would help differentiate this from `glue_job_folder_to_s3`